assets/styles.css

```diff
@@ -21,51 +26,57 @@ h3 {
   @apply text-xl;
 }
 h4 {
-  @apply text-lg
+  @apply text-lg;
 }
 h5 {
-  @apply text-base
+  @apply text-base;
 }
 h6 {
-  @apply text-sm
+  @apply text-sm;
+}
+
+.note-date {
+  @apply text-sm text-gray-500;
```

Comment:
you can add these as classes to your element, there's no need to create a class for it.

Reply:
This is a part I personally don't like about tailwind. have much class on dom elements! Why should a dom element have so many classes instead of one DRY class name?

Reply:
The project is using Tailwind and we should follow its best practices. You can use other methods in your projects.

Reply:
OK.